scripts/submit-connect-form.py

```
@@ -0,0 +1,52 @@
+STRAPI_URL = "https://backend.apps.viswam.ai"
+API_TOKEN = "YOUR_API_TOKEN"  # Replace with your Strapi API Token
+
+HEADERS = {
+    "Authorization": f"Bearer {API_TOKEN}",
+    "Content-Type": "application/json"
+}
+
+def create_connect_form(form_data):
+    """Create a connect-form entry in Strapi"""
+    url = f"{STRAPI_URL}/api/connect-forms"
+    response = requests.post(url, headers=HEADERS, json={"data": form_data})
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)